templates/create_order.html

```diff
@@ -0,0 +1,221 @@
+        body {
```

Comment:
There are too many styles in html files, maybe they should be moved to css files